.github/issue_templates/Bug_reporting.md

```diff
@@ -0,0 +1,25 @@
+    Where did it occur?
+
+### Steps to reproduce the issue
+
+    1. 
+    2. 
+    3.
+
+### Code Snippet:
+
+```python
+# Code
+```
+
+### Expected behavior (What should happen)
+
+### Actual behavior (What does happen)
+
+### Additional notes
```

Comment:
Maybe a section for affected code, processes, or data might be a helpful addition? 
```suggestion
### Additional notes

### Affected areas (code, data, or processes)
```